src/blueapi/core/device_lookup.py

```diff
@@ -48,3 +48,7 @@ def find_component(obj: Any, addr: List[str]) -> Optional[D]:
             f"Found {component} in {obj} while searching for {addr} "
             "but it is not a device"
         )
+
+
+def inject(x):
```

Comment:
Worth commenting that types are intentionally omitted to bypass type checking? This function doesn't look like it does much otherwise. I think `x` could be typed as `str` as well.

Reply:
Have added some doc string and if the linting fails I can revert the name: str